test/unit/spec/common/lang-Spec.js

```diff
@@ -0,0 +1,158 @@
+define(['common/lang'], (Utils) => {
```

Comment:
Testing the lang module all through. Some edge cases in here that could maybe be handled better, but it should all be readable enough.